source/features/pr-branch-auto-delete.tsx

```diff
@@ -45,5 +45,6 @@ void features.add(import.meta.url, {
 		onPrMerge,
 	],
 	onlyAdditionalListeners: true,
+	deduplicate: false,
```

Comment:
Felt cute, might delete later

https://user-images.githubusercontent.com/1402241/182197313-11726cf8-bb98-4da6-bebf-d17f9d464324.mov

Made a video showing our freshest features after going "back" from the commits tab